### ISSUE TITLE
Produce revised guidance for getting set-up on the Autspace's code repository

### ISSUE THREAD

**Issue by JamesSCTJ** [alan-turing-institute/AutSPACEs#75]
Current guidance for getting set-up on the Autspaces repository is unclear; particularly in respect of downloading key software components beyond Python.  This is an accessibility issue and should be resolved as quickly as possible.

**Comment by GeorgiaHCA** [comment]
Hi @JamesSCTJ - thank you for flagging this. I agree this is crucial for accessibility. I will make it a priority today.

**Comment by GeorgiaHCA** [comment]
Hey @JamesSCTJ thanks so much for your input this morning. I'm delighted you're learning python 🐍 For this issue I think screen shots of how it should look would be useful, and some troubleshooting for main issues that come up! Do you think you could procide some screenshots of where you get stuck? I think some of this has to do with setting which is different for a windows than a mac?

**Comment by jhlink** [comment]
Hey @JamesSCTJ , 

Could you describe which steps in this [readme](https://github.com/GeorgiaHCA/AutSPACEs/blob/main/dev_readme.md) that you're experiencing trouble? 

As for dependent packages, this is handled automagically depending on how you want to proceed with development. If you're developing locally, you'll need to have the appropriate Python version needed to run `poetry`, which would handle downloading all the dependencies you need. 

If you're developing with Docker, the Readme above should help you with that. And if you're experiencing any trouble, please feel free to ask any questions!
